src/compiler/prioritization-process.md

```diff
@@ -0,0 +1,197 @@
+# Prioritization Process
+
+The prioritization WG has a process that iterates every two weeks at the end of which we fill up a document with [meeting notes](https://hackmd.io/pHb6eTZ2Sjy6XZmwXZHIBA?view) of `T-compiler` working group (TODO: what is the correct cadence of these meetings?).
+
+## A typical triaging workflow
+
+#### 1. Issue gets filed on the [rust-lang repository](https://github.com/rust-lang/rust)
+
+#### 2. Release team applies labels to issues
+
+One or more "area" labels are applied to identify the area of responsability. Area labels are expressed as `A-*` (ex. "A-error-handling", "A-ffi", ... full list [here](https://github.com/rust-lang/rust/labels?q=A-))
+
+One or more "team" labels are applied to assign responsability for that issue to the relevant team(s). Team labels are expressed as `T-*` (ex. "T-cargo", "T-compiler", ... full list [here](https://github.com/rust-lang/rust/labels?q=T-))
+
+If bisection is needed, a [MCVE](https://stackoverflow.com/help/minimal-reproducible-example) is requested as well:
+
+* label `needs-bisection`, `needs-mcve` are added to the issue
+* label `ICEBreaker-Cleanup-Crew` is added to the issue (they can help improving the quality of the report, by producing a MCVE or the bisection)
+
+In general, if the scope and impact of the issue is straightforward to understand, it is directly prioritized or sent to the relevant area, otherwise it is  "nominated" for compiler team meeting (that is, appointed for discussion during the meeting).
+
+#### 3. Release team nominates for compiler team to further process (TODO: improve this section)
+
+* also cc folks to bisect
+* (usually ICE), usually includes needs-bisection and needs-mcve
+
+#### 4. Compiler team triage group analysis
+
+If the issue is found to be a critical bug, the label [P-critical](priority-levels.html#p-critical) will be applied and a discussion will happen whether the issue need to be delegated to another team.
+
+## The process in detail
+
+Summary of the important moments of the biweekly iteration:
+
+- [Drafting the meeting agenda](#drafting-the-meeting-agenda)
+- [Day before the meeting](#day-before-the-meeting)
+- [After the meeting](#after-the-meeting)
+
+We check issues for `T-compiler` and `libs-impl` and also stable/beta nominations for `T-rustdoc`.
+
+We are in the process of automating some of our work and we execute some other things in an async way. In particular, some Zulip topics are created when certain things happen to some issue.
+
+## Async process proposal (TODO: what is this section and what to do with it?)
```

Comment:
This part is copied from [this comment](https://github.com/rust-lang/compiler-team-prioritization/issues/6#issuecomment-650418768) of the issue this PR refers to.

What to do with this block? Is there some content relevant to this document?